itests/helloworld.java

```diff
@@ -1,6 +1,7 @@
 ///usr/bin/env jbang "$0" "$@" ; exit $?
 // //DEPS <dependency1> <dependency2>
 //JAVA_OPTIONS -Dfoo=bar "-Dbar=aap noot mies"
+//MANIFEST foo bar=baz baz=${bazprop:nada}
```

Comment:
split this to two lines to test that actually works.

Reply:
wdym? to test that having multiple lines works you mean?

Reply:
also if you do not add -Dbazprop the build fails.

That is somewhat of a breaking change.

but feels right we should fail - might just want to make the error message more informative.

Reply:
> wdym? to test that having multiple lines works you mean?

yeah

Reply:
wdym it fails? where does it fail? how? I don't see that here (and it shouldn't happen).

Reply:
On command Line.

if you had //JAVAAGENT key=${wonka} in a file it would pas before but today it fails as wonka key does not exist. 

More an observation and that the error message does not tell you *where* the missing key is.

Reply:
Aaah ok, but that seems like an unlikely value to have in your source file if properties are not supported.

And being able to tell _where_ is actually pretty hard right now. It would probably need an entire refactor of the parsing code. Unless there's some clever trick I'm missing.